lib/LMSManagers/LMSEventManager.php

```diff
@@ -115,6 +115,130 @@ public function GetEvent($id) {
 		return $event;
 	}
 
+    function GetEventList($year=NULL, $month=NULL, $day=NULL, $forward=0, $customerid=0, $userid=0, $type=0, $privacy=0, $closed=NULL) {
```

Comment:
Do tego zmieniasz $closed = '' na $closed = NULL, a isset($closed) dla NULL zwraca false!

Reply:
Nie rozumiem dlaczego to nie działa:

Zobacz, że:
gdy closed = '', closedfilter = ''
gdy closed = 0 lub 1, closedfilter ma wartość z warunku switch
gdy closed = NULL, zmienna closedfilter jest pusta co nie powoduje zmian w querry.

Jak powinno być właściwie?